### PR TITLE
Clarify readme

### PR DESCRIPTION
**Pull request by muescha**
changes:

- clarify which readme is used
- fix the result

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ var remark = require('remark')
 var toString = require('mdast-util-to-string')
 var Index = require('unist-util-index')
 
-// Read this readme:
+// parse the readme.md of this repo:
```
Comment (wooorm) [inline readme.md]:
```suggestion
// Parse and read this repo’s readme:
```

How about this? It still includes that the file is also read.

File: readme.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ console.log(index.get('unist').map(node => node.url))
 Yields:
 
 ```js
-[ 'Install', 'Usage', 'API', 'Related', 'Contribute', 'License' ]
+[ 'Install', 'Use', 'API', 'Related', 'Contribute', 'License' ]
```
Comment (wooorm) [inline readme.md:47]: nice, thanks for the catch!

File: readme.md (hunk 1 above)
Comment (muescha) [reply readme.md]: great :+1: :)

Comment (muescha) [reply readme.md]: updated the PR

**Comment by wooorm** [comment]
Thanks @muescha!

**Pull request merged**